content/workshops/security_openshift/exercise1.7.adoc

```diff
@@ -1,95 +1,196 @@
 ---
-title: "Exercise 1.7 - Red Hat Security API"
+title: "Exercise 1.7 - Red Hat Container Catalog"
 workshops: security_openshift
 workshop_weight: 80
 layout: lab
 ---
 
+:icons: font
 :source-highlighter: highlight.js
 :source-language: bash
 :imagesdir: /workshops/security_openshift/images
 
-= Exercise 1.7 - Red Hat Security API
+= Exercise 1.7 - Red Hat Container Catalog
 
-image::sec-api.png[]
+image::catalog.png[]
 
-Red Hat Product Security is committed to providing tools and security data to help you better understand security threats. This data has been available on our Security Data page and will now also be available in a machine-consumable format with the Security Data API. This tool will allow customers to programmatically query the API for data that was previously exposed only through files on our Security Data page.
+Red Hat Container Catalog
 
-The data provided by the Security Data API is the same as what is found on the Security Data page: OVAL definitions, Common Vulnerability Reporting Framework (CVRF) documents and CVE data. All data is available in its native XML format or in a representative JSON format.
+{{< blockquote >}}
+While public registries and uncurated repositories are acceptable for some cloud-native development and proof-of-concept projects, they do not always provide content that is fit for production consumption; enterprise workloads require enterprise-ready tools.
+{{< /blockquote >}}
 
-== Step 1:
+`MATTHEW HICKSVICE PRESIDENT, ENGINEERING, OPENSHIFT AND MANAGEMENT, RED HAT`
```

Comment:
Shouldn't this be `MATTHEW HICKS, VICE PRESIDENT, ENGINEERING, OPENSHIFT AND MANAGEMENT, RED HAT`?